src/handlers/localstorage/nonceManager.ts

```diff
@@ -0,0 +1,29 @@
+import { getGlobal, saveGlobal } from './common';
+
+export const NONCE_MANAGER = 'nonceManager';
+
+interface NetworkNonceInfo {
+  nonce: number;
+}
+interface AccountNonceInfo {
+  [key: string]: NetworkNonceInfo;
```

Comment:
can we use `[key: Network]` from `helpers/networkTypes` (this will later be moved to entities with Marco's work)

Reply:
I'm not sure an enum like `Network` can be used as an interface key.

`type AccountNonceInfo = Record<Network, NetworkNonceInfo>;` works as intended. Let me know if this is suitable.